### DISCUSSION TITLE
Etudiante -besoin d'aide

### DISCUSSION THREAD

**Discussion by Amelie80** [Security-Onion-Solutions/securityonion#13305] [Unsupported Versions]
Bonsoir à tous, je me nomme Amélie  étudiante en Réseaux et Cybersécurité. Je travail actuellent sur un projet SOC. Il s'agit de déployer S.O avec ses outils intégrés dont : Suricata, Zeek,  Elastic Stack, TheHive, Cortex, Wazuh, Sguil, Squert, CyberChef, Playbook, Stenographer et CapME.  En parallèle, je dois configurer d'autres solutions de sureveillance et sécurité dont pfSense, Grafana. 

La version utilisé ici est la 2.4.70 en architecture Standalone. 

J'ai déjà effectué l'installation sur une VM et déjà déployer un agent fleet pour l'ingestion des logs pfSense. 

Le problème ici est que je n'ai aucune idée sur comment activer ou intégrer les composants de S.O lui-meme dans la plateforme S.O installée (TheHive, Cortex, Wazuh, Sguil, Squert, Playbook, Stenographer et CapME , Grafana) . 
Si quelqu'un pourrait me porter main forte, une explication je se serais ravie. Merci d'avance.

Notons que la machine hoste (S.O) reponds complètements aux préréquis récommandés par S.O pour l'architecture Standalone.

**Comment by dougburks** [comment]
TheHive, Cortex, Wazuh, Sguil, Squert, Playbook, CapME, and Grafana have all been replaced by more modern tools. Please see the current documentation at:
https://securityonion.com/docs

**Reply by Amelie80** [reply]
Merci pour votre réponse.  Suite à votre j'ai pu trouver les infos suivantes et deduire :
Security Onion a évolué pour inclure une gamme d'outils modernes qui remplacent certaines des fonctionnalités des outils plus anciens. Voici une comparaison des outils actuels de Security Onion avec les anciens outils :

Outils Actuels dans Security Onion :
1.Alerts
   - Centralise les alertes de sécurité.
   - Comparé à : Sguil et Squert pour la gestion et l'affichage des alertes.
2. Dashboards
   - Tableau de bord pour une vue d'ensemble.
   - Comparé à : Grafana et Squert pour la visualisation des données.
3. Hunt
   - Interface pour la recherche proactive des menaces.
   - Comparé à : Sguil pour l'investigation des incidents.
4. Cases
   - Gestion des incidents et des cas.
   - Comparé à : TheHive pour le suivi des incidents.
5. Detections
   - Moteur de détection pour identifier les menaces.
   - Comparé à : Wazuh pour les capacités de détection des menaces.
6. PCAP
   - Capture et analyse des paquets.
   - Comparé à : CapME pour l'analyse des paquets.
7. Kibana
   - Visualisation et analyse des données.
   - Comparé à : Grafana pour les tableaux de bord et les visualisations.
8.ATT&CK Navigator
   - Cartographie des techniques de MITRE ATT&CK.
   - Comparé à : N/A (Spécifique à la cartographie MITRE ATT&CK).

Outils Anciens dans Security Onion :
1. TheHive
   - Gestion des incidents et des cas.
2. Cortex
   - Analyse et enrichissement des données.
3. Wazuh
   - Détection des menaces et surveillance.
4. Sguil
   - Gestion des alertes et analyse des incidents.
5. Squert
   - Interface web pour Sguil, affichage des alertes.
6. Playbook
   - Orchestration et automatisation des réponses (Playbooks).
7. CapME
   - Extraction et analyse des paquets capturés.
   
Comparaison des Fonctionnalités
- Alerts et Detections remplacent efficacement  Sguil et Squert en offrant une gestion centralisée des alertes et des capacités de détection avancées.
- Cases est comparable à TheHive, fournissant une gestion robuste des incidents.
- PCAP prend en charge les fonctions de CapME pour l'analyse des paquets réseau.
- Dashboards et Kibana remplacent Grafana et Squert pour la visualisation des données et des alertes.
- CyberChef ajoute des capacités uniques pour la manipulation des données, non directement comparables mais complémentaires.
- ATT&CK Navigator offre des capacités spécifiques de cartographie des techniques de cyberattaque, sans équivalent direct dans les outils plus anciens.

est-ce correste ?

**Reply by dougburks** [reply]
Mostly correct, but here are a few clarifications.

InfluxDB replaced Grafana for health metrics and visualization:
https://docs.securityonion.net/en/2.4/influxdb.html

Elastic Agent replaced Wazuh for endpoint visibility:
https://docs.securityonion.net/en/2.4/elastic-agent.html

Detections is a web interface for tuning NIDS, Sigma, and YARA rules:
https://docs.securityonion.net/en/2.4/detections.html

**Comment by Amelie80** [comment]
Merci énormement à vous pour cette clarification.

**Comment by Amelie80** [comment]
Vous m'avez bien aidé, merci

**Comment by Amelie80** [comment]
Bonsoir à vous, 
J'aimerais quand meme utiliser grafana pour la visualisation et autres car je ne sais pas trop mais il me semble qu'il y a une difference de fonctionnalité entre influxdb et grafana. Dans mon cas j'aimerais utiliser grafana comme outil de monitoring et visualisation des data. Par conséquant j'aimerais savoir est-il possible de relier grafana et security onion (elastiicsearch) dans cette version? et si oui toutes proposition d'integration serait les bienvenues. Merci d'avance.

**Reply by dougburks** [reply]
You may be able to manually connect Grafana but we can't provide any support for that. Most folks are fine with the influxdb configuration that comes by default in Security Onion.

**Comment by Amelie80** [comment]
d'accord merci

**Comment by Amelie80** [comment]
Bonjour,
J'effecttue un teste de configurations, j'ai ajjouter le scriptes ci-dessous dans le fichier locad.load de zeek:

protocols/snmp/main
protocols/http/main
protocols/ssl/main
protocols/rtsp/main
protocols/rtp/main
protocols/dns/main
protocols/ftp/main
protocols/syslog/main
protocols/smb/main
protocols/dce-rpc/main
protocols/udp/main

depuis je rencontres le meme soucis : 
![image](https://github.com/user-attachments/assets/c4f4d04e-f012-4f8c-9053-c197df1bf43d)

![SO2](https://github.com/user-attachments/assets/92921032-553a-4b31-9ebd-f9bd1f62d0aa)

apres quelque mininute ça donne cà : 
![image](https://github.com/user-attachments/assets/6ed6abd5-41fc-4437-b835-710f91d0ae6e)
 et ensuite il redevient comme cela : 
![image](https://github.com/user-attachments/assets/92f222d8-1747-4978-88de-0f324171ca59)

lorsque je viens sur l'interface grid pour verifier l'etat de SO et ses composants, il me met missing pour zeek

**Reply by dougburks** [reply]
Our default Zeek configuration out of the box should support most major protocols. Please revert your custom Zeek configuration.
